packages/govern-subgraph/schema.graphql

```diff
@@ -86,13 +84,8 @@ type Action @entity {
   data: Bytes!
 }
 
-type Schedule @entity {
-  id: ID!
-  container: Container!
-  collateral: Collateral!
-  createdAt: BigInt!
-}
-
+# I honestly think this should not be here,
+# but we can handle this later.
 type Challenge @entity {
```

Comment:
As I say here, I really think that this entity should not exist here. It seems out of scope for the subgraph itself because if an action gets disputed, it's really up to the dispute resolution protocol to be able to get this info from its own graph; we should only facilitate finding the actual disputed action so that they're able to show context regarding it.

Removing this required some hefty changes from the subgraph, so I avoided it, but definitely I think at most, we should just set it to the actual `Challenged` state, and on the `history` field provide some more info on the `data` field.